src/pages/Home.vue

```diff
@@ -20,12 +20,35 @@
             </div>
         </div>
 
-        <div class="container card card-body py-4">
+        <div class="container card card-body py-4 my-4">
             <h4>What is the Mappers' Guild?</h4>
             <div class="text-secondary">
                 <a href="https://osu.ppy.sh/wiki/Community/Mappers_Guild" target="_blank">The Mappers' Guild</a> is a hub for beatmaps associated with <a href="https://osu.ppy.sh/beatmaps/artists" target="_blank">osu!'s Featured Artists</a>!
             </div>
             <hr />
+            <div>For the most newest info about Featured Artist mapping, check out the channels below on the <a href="https://discord.gg/ppy" target="_blank">osu!dev Discord server</a>.</div>
+            <ul>
+                <li>
+                    <code>#featured-artists</code>
+                    <ul>
+                        <li>Events related to Featured Artists. Currently, this involves creating <a href="https://discord.com/channels/188630481301012481/1094639474941636819/1094685118603481129" target="_blank">playlists for osu!(lazer) with new Featured Artist beatmaps!</a></li>
+                        <li>Quick feedback from the mapping community on your Featured Artist maps</li>
+                        <li>Discussion related to Featured Artists</li>
```

Comment:
```suggestion
                        <li>Discussion and events related to Featured Artists, such as creating <a href="https://discord.com/channels/188630481301012481/1094639474941636819/1094685118603481129" target="_blank">playlists for osu!(lazer) with new Featured Artist beatmaps!</a></li>
                        <li>Quick feedback from the mapping community on your Featured Artist maps</li>
```
feels weird to have these be separate points given they are similar and both use the phrase "related to featured artists". i adjusted the playlist wording to account for this